docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -68,6 +68,9 @@ You do *not* need to include the underlying `spring-cloud-gcp-pubsub` dependency
 
 A summary of these artifacts are provided below.
 
+Aside from these modules, we also provide additional starters with auto-configurations to various Google Client Libraries.
+Refer for the full list https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/main/spring-cloud-previews/README.md[here].
```

Comment:
It might be possible to construct a URL based on `{project-version}`, like:
`https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/v{project-version}/spring-cloud-previews/README.md`.
It's just a potential alternative for generating the list directly into the docs. However, it would need to be conditional on the project version being non-snapshot. So, let's not do that now.